vec/src/Data/Vec/Pull.hs

```diff
@@ -380,6 +388,18 @@ ifoldr f z (Vec v) = I.foldr (\a b -> f a (v a) b) z F.universe
 foldl' :: N.SNatI n => (b -> a -> b) -> b -> Vec n a -> b
 foldl' f z (Vec v) = I.foldl' (\b a -> f b (v a)) z F.universe
 
+scanr :: forall a b n. N.SNatI n => (a -> b -> b) -> b -> Vec n a -> Vec ('S n) b
+scanr f z = fromJust . fromList . NonEmpty.toList . NonEmpty.scanr f z
```

Comment:
`fromJust`, why? This cannot be right.

Reply:
I'd rather not add scans for `Pull` variant. They barely make any sense for this representation.

Reply:
Removed!